File: scripts/db_tools/check_db.py
import urllib.request
import json
url = 'https://organic-reason-4ln7n-default-rtdb.firebaseio.com/users.json'
try:
    with urllib.request.urlopen(url) as response:
        data = json.loads(response.read().decode())
        keys = [k for k in data.keys() if 'dammam' in k.lower()]
        for k in keys:
            print(k)
            print(json.dumps(data[k], indent=2))
except Exception as e:
    print('Error:', e)

File: scripts/db_tools/check_db.js
const fetch = require('node-fetch');
const url = 'https://automation-suit-cece7-default-rtdb.firebaseio.com/users.json';
fetch(url).then(r => r.json()).then(data => {
  const keys = Object.keys(data).filter(k => k.toLowerCase().includes('dammam'));
  keys.forEach(k => console.log(k, JSON.stringify(data[k], null, 2)));
}).catch(console.error);
